### PR TITLE
FXML-1067 Bump onnx-mlir to latest main, llvm to commit '59548fe873d8d98e359fb21fbb2a0852fed17ff5'

### PR DESCRIPTION
**Pull request by gargaroff**

### CODE REVIEW

File: test/mlir/conversion/onnx_to_torch/pad/padV2_noop.onnx.mlir [hunk 1]
```diff
@@ -1,10 +1,10 @@
 //RUN: onnx-mlir --EmitONNXIR --run-torch-pass %s -o=%t >/dev/null && cat %t.onnx.mlir | FileCheck -v %s
 
 module {
-  func.func @main_graph(%arg0: tensor<20x16x44x32xf32>) -> tensor<20x16x48x38xf32> attributes {input_names = ["0"], output_names = ["2"]} {
+  func.func @main_graph(%arg0: tensor<20x16x44x32xf32>) -> tensor<20x16x44x32xf32> attributes {input_names = ["0"], output_names = ["2"]} {
```
Comment (ljfitz) [inline test/mlir/conversion/onnx_to_torch/pad/padV2_noop.onnx.mlir:4]: Thanks for fixing these.

**Review by ljfitz — COMMENTED** [review]
We have unnecessary changes in `.gitignore` and `src/Transform/ONNX/CMakeLists.txt`. Could you please revert those to the upstream contents?

**Comment by gargaroff** [comment]
> We have unnecessary changes in `.gitignore` and `src/Transform/ONNX/CMakeLists.txt`. Could you please revert those to the upstream contents?

I see a whitespace change in CMakeLists.txt that I will revert. Are you referring to that or do you also see other changes?
Also what changes are you seeing in `.gitignore`? I don't even see it in the list of changed files.

File: test/mlir/conversion/onnx_to_torch/pad/padV2_noop.onnx.mlir (hunk 1 above)
Comment (gargaroff) [reply test/mlir/conversion/onnx_to_torch/pad/padV2_noop.onnx.mlir:4]: Only cost us two days of finding a bug that didn't exist 😬

**Review by ljfitz — APPROVED** [review]

**Comment by ljfitz** [comment]
> > We have unnecessary changes in `.gitignore` and `src/Transform/ONNX/CMakeLists.txt`. Could you please revert those to the upstream contents?
> 
> I see a whitespace change in CMakeLists.txt that I will revert. Are you referring to that or do you also see other changes? Also what changes are you seeing in `.gitignore`? I don't even see it in the list of changed files.

`git diff $(git merge-base origin/main origin/dominik.bump_llvm.FXML-1067) origin/dominik.bump_llvm.FXML-1067 -- .gitignore`:
```
diff --git a/.gitignore b/.gitignore
index 6a65a79b..e1050000 100644
--- a/.gitignore
+++ b/.gitignore
@@ -186,3 +186,16 @@ dmypy.json
 
 # Visual Studio Code Files
 .vscode
+
+# Gnu Global
+GPATH
+GRTAGS
+GTAGS
+
+# Emacs save file
+
+\#*\#
+
+# MLIR files
+*.onnx.onnx.mlir
+*.tmp
\ No newline at end of file
```

**Comment by gargaroff** [comment]
@ljfitz I took the file content of the .gitignore from our `feature/onnx-to-torch` branch and compared it to my local version that is in this commit, and it is identical. I'm wondering where the diff is coming from. I do however see that it is different to the version on the main branch.

**Pull request merged**